probe-rs/src/probe/wlink/mod.rs

```diff
@@ -491,7 +491,7 @@ impl JtagAccess for WchLink {
                     | (op as u128);
 
                 let ret_bytes = ret.to_le_bytes();
-                let mut ret = bitvec![0;32];
+                let mut ret = bitvec![0;0];
```

Comment:
I'd rather turn this into `BitVec::with_capacity(32)`.

Reply:
Done but with 128 as that's the returned size